src/daria/corrections/color/colorcorrection.py

```diff
@@ -169,6 +199,17 @@ def __call__(
         # Ensure a data format which can be used by cv2 etc.
         return corrected_image.astype(np.float32)
 
+    def write_config_to_file(self, path: Path) -> None:
```

Comment:
I am a much bigger fan of `Path` than `str`. I am trying to not use the latter if possible. However, we usually allow for both. Why not here?